rpm_spec/subpackages/manageiq-appliance

```diff
@@ -109,4 +109,5 @@ fi
 %{_unitdir}/manageiq-messaging-ready.service
 %{_unitdir}/miqtop.service
 %{_unitdir}/miqvmstat.service
+%{_sysconfdir}/default/manageiq-appliance.properties
```

Comment:
Should this live up with the other _sysconfdir entries 10-ish lines up?

Reply:
Sure